src/main/java/com/epam/repairstuidiospring/service/repository/impl/OrderRepositoryImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.repairstuidiospring.service.repository.impl;
+
+import com.epam.repairstuidiospring.service.model.Order;
+import com.epam.repairstuidiospring.service.model.User;
+import com.epam.repairstuidiospring.service.repository.OrderRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers